lib/inspec/profile.rb

```diff
@@ -128,7 +128,8 @@ def initialize(source_reader, options = {})
         cli_input_files: options[:runner_conf][:input_file], # From CLI --input-file
         profile_metadata: metadata,
         # TODO: deprecation checks here
-        runner_api: options[:runner_conf][:attributes] # This is the route the audit_cookbook and kitchen-inspec take
+        runner_api: options[:runner_conf][:attributes], # This is the route the audit_cookbook and kitchen-inspec take
+        cli_input_arg: options[:runner_conf][:input]
```

Comment:
this diff is why I like to leave trailing commas... but I've not gotten used to doing it in arg lists.

Reply:
I like trailing commas too, but chefstyle does not permit them.